apps/vr-tests/.storybook/preview.js

```diff
@@ -27,16 +29,37 @@ const defaultConfig = {
  */
 setAddon({
   addStory(storyName, storyFn, config = defaultConfig) {
-    this.add(storyName, context => {
-      setRTL(false);
-      return storyFn(context);
-    });
-
-    if (config.rtl) {
-      this.add(storyName + ' - RTL', context => {
-        setRTL(true);
+    // V-Next stories
+    if (this.kind.includes('Next') || config.provider === 'FluentProvider') {
+      this.add(storyName, context => {
+        return (
+          <FluentProvider theme={webLightTheme} dir={'ltr'}>
+            {storyFn({ context })}
+          </FluentProvider>
+        );
+      });
+      if (config.rtl) {
```

Comment:
Can we rename this to something more descriptive? For example, `testAlsoInRtl` or `includeRtl`?

```tsx
.addStory('Root', () => <Label>I'm a label</Label>, { rtl: true })
```

^ current naming is a bit confusing as it looks that a story will be tested _only_ in RTL

Reply:
Makes sense to me. It would probably be best to do that in a separate pull request after this since previous stories also call it rtl and would raise the scope here quite a bit.

Reply:
I like `includeRtl` though it's probably not super high priority

Reply:
Here is a pr for renaming it to `includeRtl`: https://github.com/microsoft/fluentui/pull/19792